pa11ycrawler/spiders/edx.py

```diff
@@ -10,8 +11,28 @@
 from scrapy.linkextractors import LinkExtractor
 from pa11ycrawler.items import A11yItem
 
+LOGIN_HTML_PATH = "/login"
+LOGIN_API_PATH = "/user_api/v1/account/login_session/"
 AUTO_AUTH_PATH = "/auto_auth"
-COURSE_BLOCKS_API_PATH = "/api/courses/v1/blocks"
+COURSE_BLOCKS_API_PATH = "/api/courses/v1/blocks/"
+
+
+def get_csrf_token(response):
+    """
+    Extract the CSRF token out of the "Set-Cookie" header of a response.
+    """
+    cookie_headers = [
+        h.decode('ascii') for h in response.headers.getlist("Set-Cookie")
+    ]
+    if not cookie_headers:
+        return None
+    csrf_headers = [
+        h for h in cookie_headers if h.startswith("csrftoken=")
```

Comment:
Generally I would agree with you, but in this case I think it makes sense. The problem with one-character variables is that it's easy to forget what the name means. However, because this variable is literally declared and used only on one line, there's no chance of forgetting what the name means. I kept the variable name short to prevent the line from getting too long.

Even if pylint warns about this, I would just silence the warning. The purpose is clarity, and I believe this is plenty clear.